index.html

```diff
@@ -1,19 +1,147 @@
+      href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@300&family=Montserrat:wght@200&family=Work+Sans:wght@200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <a class="header-logo" href="#">
+          <img
+            class="logo-image"
+            src="./img/karma-logo.svg"
+            alt="logo of karma"
+          />
+        </a>
+        <nav class="nav">
+          <!-- (B) THE HAMBURGER -->
+          <label for="hamburger">&#9776;</label>
+          <input type="checkbox" id="hamburger" />
+          <ul class="nav-menu">
+            <li class="list-item">
+              <a href="./index.html" class="nav-link nav-active">Meet Karma</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">How it Works</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Store</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Help</a>
+            </li>
+            <li class="list-item">
+              <a href="./order.html" class="nav-link">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="home">
+        <div class="content">
+          <h2 class="intro-karma">Introducing Karma</h2>
+          <p class="slogan">Bring WiFi with you everywhere you go.</p>
+          <button class="learn-more">Learn More</button>
+        </div>
+      </section>
+      <section class="products">
+        <h2>Everyone needs a little Karma.</h2>
+        <div class="product-cards">
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-devices.svg"
+              alt="devices-logo"
+            />
+            <p>Internet for all devices</p>
+          </article>
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-coffee.svg"
+              alt="coffee-logo"
+            />
+            <p>Boost your productivity</p>
+          </article>
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-refill.svg"
+              alt="refill-logo"
+            />
+            <p>Pay as you go</p>
+          </article>
+        </div>
+      </section>
+      <section class="quoto-section">
+        <img
+          class="quoto-image"
+          src="./img/homepage-feature.jpg"
+          alt="quoto image"
+        />
+        <article class="quoto-container">
+          <div class="quoto-content">
```

Comment:
What is this `div` doing that `.quoto-container` couldn't do? They are wrapping exactly the same elements. It looks like you could remove the div and just have the article (which by the way could be a `blockquote` element instead)